tajo-plan/src/main/java/org/apache/tajo/plan/logical/ProjectionNode.java

```diff
@@ -64,8 +65,13 @@ public void setTargets(List<Target> targets) {
   public List<Target> getTargets() {
     return this.targets;
   }
-	
-	public void setChild(LogicalNode subNode) {
+
+  @Override
+  public Stream<Target> targets() {
```

Comment:
This is because we still need List allowing access elements by index in some codes.